velox/exec/tests/TableWriteTest.cpp

```diff
@@ -93,6 +94,39 @@ class TableWriteTest : public HiveConnectorTestBase {
     return getRecursiveFiles(directoryPath).size();
   }
 
+  // Returns all available table types to test insert without any
+  // partitions (used in "immutablePartitions" set of tests).
+  std::vector<connector::hive::LocationHandle::TableType> tableTypes() {
+    return {
+        connector::hive::LocationHandle::TableType::kNew,
```

Comment:
There is a temp table type too that is not supported. Please explicitly state that.